app/controllers/claims/support/claims/edit_request_clawback_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Claims::Support::Claims::EditRequestClawbackController < Claims::ApplicationController
+  include WizardController
+  before_action :skip_authorization
+  before_action :set_claim
+  before_action :set_wizard
+  helper_method :index_path
+
+  def new
+    @wizard.setup_state
+    redirect_to step_path(@wizard.first_step)
+  end
+
+  def update
+    if !@wizard.save_step
+      render "edit"
+    else
+      @wizard.update_clawback
+      @wizard.reset_state
+      redirect_to index_path, flash: {
+        heading: t(".success_heading"),
+      }
+    end
+  end
+
+  private
+
+  def set_claim
+    @claim = Claims::Claim.find(params[:claim_id])
+  end
+
+  def set_wizard
+    state = session[state_key] ||= {}
+    current_step = params[:step]&.to_sym
+    @wizard = Claims::EditRequestClawbackWizard.new(claim: @claim, params:, state:, mentor_training_id:, current_step:)
+  end
+
+  def step_path(step)
+    edit_request_clawback_claims_support_claims_clawback_path(claim: @claim, mentor_training_id:, state_key:, step:)
+  end
+
+  def mentor_training_id
+    params[:mentor_training_id]
```

Comment:
If this is expected to be there, you should use `fetch`:
```suggestion
    params.fetch(:mentor_training_id)
```
`fetch` will error if the key isn't there.